js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,21 @@
+function updateObject(index, newValue, array) {
+  if (Number.isNaN(parseInt(index, 10))) {
+    throw new Error(`Expected a number, got ${typeof index}`);
+  }
+  if (!Array.isArray(array)) {
+    throw new Error(`Expected an array, got ${typeof array}`);
+  }
+  const arrayLength = array.length;
+  const absoluteIndex = Math.abs(index);
+  if ((index < 0 && absoluteIndex > arrayLength)
```

Comment:
Break this condition by using suitable names such as `const indexOutOfBounds = absoluteIndex > arrayLength`